### PR TITLE
Update App Store screenshots + whatsnew

### PR DESCRIPTION
**Pull request by TheoChevalier**
Do not merge yet
https://bugzilla.mozilla.org/show_bug.cgi?id=1255850
https://bugzilla.mozilla.org/show_bug.cgi?id=1252410

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -414,6 +414,7 @@
     'whatsnew/whatsnew_android_44.lang',
     'whatsnew/whatsnew_android_45.lang',
     'whatsnew/whatsnew_android_46_beta.lang',
+    'whatsnew/whatsnew_ios_2_1.lang',
```
Comment (flodolo) [inline app/config/sources.inc.php]: Next version of iOS is 3.0, not sure where the 2.1 comes from.

Comment (TheoChevalier) [inline app/config/sources.inc.php]: It comes from the App Store, that's the version that has just been released

Comment (flodolo) [inline app/config/sources.inc.php]: Ah, I thought it was for the version we're localizing now. Also completely missed that the minor update for 2.0 was called 2.1

**Comment by pascalchevrel** [comment]
I put the screenshots for v3.0 in a separate lang file so this invalidates this PR, I am going to close it.

**Pull request closed without merging**